arelle/PluginManager.py

```diff
@@ -143,25 +153,41 @@ def close():  # close all loaded methods
 
 '''
 
+
+def logPluginTrace(message: str, level: Number) -> None:
+    """
+    If plugin trace file logging is configured, logs `message` to it.
+    Otherwise, only logs to controller logger if log is an error.
+    :param message: Message to be logged
+    :param level: Log level of message (e.g. logging.INFO)
+    """
+    global pluginTraceFileLogger
+    if pluginTraceFileLogger:
+        pluginTraceFileLogger.log(level, message)
+    elif level >= logging.ERROR:
```

Comment:
I don't think the plugin trace file should cause errors to be removed from the cntrl. I'd make this an `if` instead of an `elif`.